context/app/static/js/components/detailPage/ProcessedData/ProcessedDataset/ProcessedDataset.tsx

```diff
@@ -45,16 +45,18 @@ function ProcessedDatasetDescription() {
   return <LabelledSectionText label="Description">{description}</LabelledSectionText>;
 }
 
-function RegisteredBy() {
+function Contact() {
   const {
-    dataset: { created_by_user_displayname, created_by_user_email },
+    dataset: { mapped_consortium },
   } = useProcessedDatasetContext();
-  return (
-    <LabelledSectionText label="Registered By">
-      {created_by_user_displayname} <br />
-      <EmailIconLink email={created_by_user_displayname}>{created_by_user_email}</EmailIconLink>
-    </LabelledSectionText>
-  );
+  if (mapped_consortium === 'HuBMAP') {
+    return (
+      <LabelledSectionText label="Contact" iconTooltipText="This is the contact for this data.">
+        <ContactUsLink>HuBMAP Help Desk</ContactUsLink>
+      </LabelledSectionText>
+    );
+  }
+  return null;
```

Comment:
Prefer the guard clause pattern returning null if the condition isn't met, but not a big deal.